pyteck/eval_model.py

```diff
@@ -40,7 +40,7 @@ def create_simulations(dataset, properties):
     Returns
     -------
     simulations : list
-        List of :class:`Simulation` objects for each simulation
+        List of :class:`AutoignitionSimulation` objects for each simulation
```

Comment:
Perhaps I should wait to review until refactoring finished, but this file/module/class seems a little confused right now about if it's specific to Autoignition, or if it's generic for all simulations. 

So I'm going to pause reviewing now. Let me know if/when I should take another look.